### ISSUE TITLE
Error applying to a folder.

### ISSUE THREAD

**Issue by mikhail-khodorovskiy** [terraform-google-modules/terraform-google-iam#18]
```
module "folder_iam_binding" {
  source = "terraform-google-modules/iam/google"
  mode   = "authoritative"

  folders = ["${google_folder.enironment.id}"]

  bindings = {

    "roles/editor" = [
      "${var.sysadmins}"
    ]

    "roles/viewer" = [
      "${var.developers}"
    ]

    "roles/logViewer" = [
      "${var.developers}"
    ]

    "roles/storageObjectViewer" = [
      "${var.developers}"
    ]

  }
}
```
produces an error:
* module.janus_environment_folder.module.folder_iam_binding.google_folder_iam_binding.folder_iam_authoritative[2]: 1 error(s) occurred:

* google_folder_iam_binding.folder_iam_authoritative.2: Error retrieving IAM policy for folder "folders/folders/999577761704": googleapi: got HTTP response code 404 with body: <!DOCTYPE html>

**Comment by dmyerscough** [comment]
I bumped into the same issue, I created the following pull request #19

**Comment by aaron-lane** [comment]
Fixed by #19.
